### PR TITLE
Update CVE-2020-7383.json

### PR DESCRIPTION
**Pull request by djordan-r7**

### CODE REVIEW

File: 2020/7xxx/CVE-2020-7383.json [hunk 1]
```diff
@@ -1,18 +1,92 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
     "CVE_data_meta": {
+        "ASSIGNER": "cve@rapid7.con",
+        "DATE_PUBLIC": "2020-10-15T11:23:00.000Z",
```
Comment (todb-r7) [inline 2020/7xxx/CVE-2020-7383.json]: Is this date (in the future) correct?

Comment (todb-r7) [reply 2020/7xxx/CVE-2020-7383.json]: yeah this wants to be Oct 14, if you please, just going by the release note date.

Comment (djordan-r7) [reply 2020/7xxx/CVE-2020-7383.json]: @todb-r7 What was the rule for the date made public? Was it when it was reported to us? I put tomorrow's date as that's when the release notes will be going out including the update on this fix.

File: 2020/7xxx/CVE-2020-7383.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "CVE_data_meta": {
         "ASSIGNER": "cve@rapid7.con",
-        "DATE_PUBLIC": "2020-10-15T11:23:00.000Z",
+        "DATE_PUBLIC": "2020-10-14T11:23:00.000Z",
```
Comment (djordan-r7) [inline 2020/7xxx/CVE-2020-7383.json:4]: @todb-r7 Changed the date to the 14th! :D

**Pull request merged**